mason/mason.go

```diff
@@ -204,6 +311,9 @@ func getSchema(schemaId string, all bool, output string) {
 	if schemaId != "" {
 		q.Add("schema_id", schemaId)
 	}
+	if projectId != "" {
```

Comment:
Please do a check for type as well as null

Reply:
By default from Flagset we will get this as `""` if it is null